.gitmodules

```diff
@@ -7,8 +7,8 @@ fxDONOTUSEurl = https://github.com/NCAR/fates-release
 
 [submodule "cism"]
 path = components/cism
-url = https://github.com/mvertens/CISM-wrapper
-fxtag = 34339777afa7b810fd4822fce1039826668b33e4
+url = https://github.com/ESCOMP/CISM-wrapper
+fxtag = cismwrap_2_2_001
```

Comment:
I'm puzzled about why these changes appear in this commit but do not appear when I "show all changes"

Reply:
...because this is not the new tag :-)